### PR TITLE
chore: improve devimint api

### PR DESCRIPTION
**Pull request by Maan2003**

### CODE REVIEW

File: devimint/src/federation.rs [hunk 1]
```diff
@@ -32,17 +32,19 @@ use super::vars::utf8;
 use crate::util::{poll, FedimintdCmd};
 use crate::{poll_eq, vars};
 
+#[derive(Clone)]
 pub struct Federation {
```
Comment (dpc) [inline devimint/src/federation.rs:36]: Does it make sense to clone this? Isn't it more like a ... resource?

Comment (Maan2003) [reply devimint/src/federation.rs:36]: all daemon structs are clone in devimint

Comment (dpc) [reply devimint/src/federation.rs:36]: We don't do things like "kill on `Drop`" etc. on them?

**Review by dpc — APPROVED** [review]

File: devimint/src/federation.rs (hunk 1 above)
Comment (Maan2003) [reply devimint/src/federation.rs:36]: that happens in ProcessHandle, which is Arc<ProcessHandleInner>

**Review by elsirion — APPROVED** [review]

**Pull request merged**